src/status_im2/contexts/onboarding/enable_biometrics/view.cljs

```diff
@@ -5,7 +5,8 @@
             [status-im2.contexts.onboarding.enable-biometrics.style :as style]
             [utils.i18n :as i18n]
             [status-im2.contexts.onboarding.common.background.view :as background]
-            [utils.re-frame :as rf]))
+            [utils.re-frame :as rf]
+            [status-im.multiaccounts.biometric.core :as biometric]))
```

Comment:
is it possible that we move the relevant biometric code in this pr to status_im2?

Reply:
it doesn't have to be the subs, but at least the helper functions 👍